src/main/java/org/prebid/server/privacy/gdpr/vendorlist/VendorListServiceV1.java

```diff
@@ -27,6 +27,7 @@
 
     public VendorListServiceV1(String cacheDir,
                                String endpointTemplate,
+                               boolean deprecated,
```

Comment:
Change arg order as in parent `org.prebid.server.privacy.gdpr.vendorlist.VendorListService` class.